common/src/main/java/io/netty/util/concurrent/AbstractEventExecutor.java

```diff
@@ -180,4 +182,17 @@ public void remove() {
             throw new UnsupportedOperationException("read-only");
         }
     }
+    
+    @Override
+    public void setRejectedTaskHandler(RejectedTaskHandler rejectedTaskHandler) {
+        if (rejectedTaskHandler == null) {
```

Comment:
You can use `ObjectUtil` for this and so eliminate some code for the null checks.